README.md

```diff
@@ -67,10 +67,18 @@ CMD dockerize -wait tcp://${NEW_RELIC_DAEMON_ADDRESS} \
   && ./ex_container.out
 ```
 
-### Daemon Container Configuration
-The daemon `Dockerfile` below shows how to configure a daemon container:
+## Daemon Container
+There are two ways to configure the daemon container:
+* Pre-made `newrelic/c-daemon` Docker image.
+* Create your own daemon container.
+
+### Daemon Docker Image
+To simplify the daemon container configuration, New Relic has published a `newrelic/c-daemon` docker image. To use it, pull the image with the `docker pull newrelic/c-daemon` command and run it with the `docker run --name [daemon-name] newrelic-daemon`. The daemon in this image listens on port `31339`. For more information on this image, visit [Docker Hub](https://hub.docker.com/r/newrelic/c-daemon) or our [repo](https://github.com/newrelic/newrelic-c-daemon-docker).
```

Comment:
Since we're using a docker-compose file here let's also say "If you're using docker-compose just add the following line `image: newrelic/c-daemon` to your `docker-compose.yml` file. If you aren't using docker-compose you can pull the image with [what you said here]"

Reply:
Nvm, you mention it below!